docs/reference/troubleshooting/discovery-issues.asciidoc

```diff
@@ -118,3 +119,79 @@ include::network-timeouts.asciidoc[tag=troubleshooting-network-timeouts-threads]
 If a node joins the cluster but {es} determines it to be faulty then it will be
 removed from the cluster again. See <<cluster-fault-detection-troubleshooting>>
 for more information.
+
+[discrete]
+[[discovery-master-split-brain]]
+=== Multiple masters are elected
+
+Infrequently, {es} can enter a {wikipedia}/Split-brain_(computing)[split-brain] 
+state, where nodes within a cluster disagree on which node is elected 
+as master node. This is a non-viable state for the cluster, and can lead to 
+inconsistencies and data loss. 
+
+To prevent state from occurring, we recommend the following practices:
+
+* <<modules-discovery-voting,Avoid even 
+master-eligible node counts>> by creating an odd number of master-eligible nodes or adding a voting-only <<high-availability-cluster-design-two-nodes, tiebreaker>>
```

Comment:
We definitely do not recommend this, indeed Elasticsearch has [specific optimizations for handling the case where there are an even number of master-eligible nodes](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery-voting.html#_even_numbers_of_master_eligible_nodes).

Reply:
Kindly port over to [cloud#128569](https://github.com/elastic/cloud/issues/128569) for a sub-discussion to compare against company-private code's recommendations 🙏

Reply:
Now that the cloud discussion is completed as tiebreakers being desirable still, can we review this thread? Will you kindly help clarify which part of the bullet is concerning and not recommended? Sorry I don't think I had understood 🙂.

Reply:
We do not recommend creating an odd number of master-eligible nodes for any reason, nor does the parity of the number of master-eligible nodes have any impact on whether split-brain might or might not occur.